scripts/deployment/upload_model_to_huggingface.py

```
@@ -0,0 +1,1252 @@
+    emotion_labels = load_emotion_labels_from_model(model_path)
+
+    # Create label mappings
+    id2label = dict(enumerate(emotion_labels))
+    label2id = {label: i for i, label in enumerate(emotion_labels)}
+
+    if os.path.isdir(model_path):
+        # Already a HuggingFace directory - copy and update
+        print("📁 Processing HuggingFace model directory...")
+
+        # Copy all files
+        for file in os.listdir(model_path):
```

Ruff (0.12.2)

Use `pathlib.Path.iterdir()` instead.

(PTH208)